src/pages/WorldEditor/WorldEditor.tsx

```diff
@@ -40,30 +35,22 @@ export const WorldEditor: React.FC = () => {
   }
 
   const editMode = !!world?.id;
-  const worldName = world?.name ?? "";
-  const adminTitle = editMode
-    ? worldName
-      ? `${worldName} settings`
-      : "Settings"
-    : "Create a new world";
+  const adminTitle = editMode ? "Settings" : "Create world";
 
   const WorldEditorPanel = PANEL_MAP[selectedTab] ?? <></>;
 
+  const crumbtrail = editMode
+    ? ALWAYS_EMPTY_ARRAY
+    : [{ name: "Switch World", href: "switch-world" }];
```

Comment:
`useMemo` because of  this?

Is the array and object inside it an initial value to be modified?

I think several options are available depending on the intended use:
- put a constant at the top of the file, best if having `Object.freeze([ Object.freeze({ ... }) ])`
- put in `useState()`
- made a return value from a factory function that always produces fresh array ready to be modified
- any combo of the above

Reply:
thanks, missed that somehow. Fixed here: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0

moved that object out of the component.